GitCommands/Git/Executable.cs

```diff
@@ -107,12 +110,12 @@ private sealed class ProcessWrapper : IProcess
                         StandardOutputEncoding = outputEncoding,
                         StandardErrorEncoding = errorEncoding,
                         FileName = fileName,
-                        Arguments = arguments,
+                        Arguments = prefixArguments + arguments,
```

Comment:
Use string interpolation
```suggestion
                        Arguments = $"{prefixArguments}{arguments}",
```

Reply:
ok